src/components/RadioInput/useRadioInput.tsx

```diff
@@ -1,34 +1,41 @@
 import React, { useState } from 'react';
 import { Styles } from './styles';
 
-function useRadioInput(
-  defaultChecked?: boolean,
-) {
+/**
+ * Custom hook for radio buttons.
+ * @param {boolean} defaultChecked If checkbox should be checked by default.
+ * @returns {any} a component, the state and function to update state.
```

Comment:
This is a really good practice, but since we are using typescript and it's intelisense is pretty smart we may not need to add the param type or return type to JS-doc. Might be best to only use the descriptions and then only add the types in the function definition itself.